lib/bot.js

```diff
@@ -95,7 +95,7 @@ Bot.prototype.run = function () {
   var self = this,
       verbose = self.config.verbose,
       bot = new slackbot(this.config.token),
-      pattern = "(?:[^a-zA-Z0-9_/-]|^)((";
```

Comment:
https://github.com/gsingers/slack-jira-plugin/pull/38 This pattern is used to reject matching on issues where the full URL is already given.  e.g. if a user puts in https://my.jira.com/FOO-18, we don't then want to link FOO-18 again.